ui/components/app/recovery-phrase-reminder/recovery-phrase-reminder.js

```diff
@@ -35,27 +35,25 @@ export default function RecoveryPhraseReminder({ onConfirm, hasBackedUp }) {
         paddingLeft={4}
         className="recovery-phrase-reminder"
       >
-        <Typography
+        <Text
           color={TextColor.textDefault}
           align={TEXT_ALIGN.CENTER}
-          variant={TypographyVariant.paragraph}
-          boxProps={{ marginTop: 0, marginBottom: 4 }}
+          variant={TextVariant.bodyMd}
+          marginBottom={4}
         >
           {t('recoveryPhraseReminderSubText')}
-        </Typography>
+        </Text>
         <Box marginTop={4} marginBottom={8}>
           <ul className="recovery-phrase-reminder__list">
-            <li>
-              <Typography
-                as="span"
-                color={TextColor.textDefault}
-                fontWeight={FONT_WEIGHT.BOLD}
-              >
-                {t('recoveryPhraseReminderItemOne')}
-              </Typography>
-            </li>
-            <li>{t('recoveryPhraseReminderItemTwo')}</li>
-            <li>
+            <Text
+              as="li"
+              color={TextColor.textDefault}
+              fontWeight={FONT_WEIGHT.BOLD}
+            >
+              {t('recoveryPhraseReminderItemOne')}
+            </Text>
+            <Text as="li">{t('recoveryPhraseReminderItemTwo')}</Text>
+            <Text as="li">
```

Comment:
```suggestion
           <li>
              <Text
                as="span"
                color={TextColor.textDefault}
                fontWeight={FontWeight.bold}
              >
                {t('recoveryPhraseReminderItemOne')}
              </Text>
            </li>
            <li>{t('recoveryPhraseReminderItemTwo')}</li>
            <li>
```

@georgewrmarshall Maybe i'm wrong here, it seems like we should keep the original structure, not sure if Text component will compile in the end. Can you double check this one?

Reply:
Makes sense, thanks for explaining, let's follow that !